tests/NHSD.BuyingCatalogue.API.IntegrationTests/Features/NativeDesktop/NativeDesktopMemoryAndStorage/Supplier Edit Native Desktop MemoryAndStorage Validation.3620.feature

```diff
@@ -0,0 +1,72 @@
+        | Sln1     | An full online medicine system | Online medicine 1 | { "ClientApplicationTypes": [], "BrowsersSupported": [], "MobileResponsive": null, "Plugins": null, "MinimumConnectionSpeed": null, "MinimumDesktopResolution": null, "HardwareRequirements": null, "NativeMobileHardwareRequirements": null, "NativeDesktopHardwareRequirements": null, "AdditionalInformation": null, "MobileFirstDesign": null, "NativeMobileFirstDesign": null, "MobileOperatingSystems": null, "MobileConnectionDetails": null, "MobileMemoryAndStorage": null, "MobileThirdParty": null, "NativeMobileAdditionalInformation": null, "NativeDesktopOperatingSystemsDescription": null, "NativeDesktopMinimumConnectionSpeed": null } |
+    When a PUT request is made to update the native-desktop-memory-and-storage section for solution Sln1
+        | MinimumMemoryRequirement | StorageRequirementsDescription | MinimumCpu   | RecommendedResolution |
+        | 512TB                    | Some description               | <FieldValue> | 800x600               |
+    Then a response status of 400 is returned
+    And the minimum-cpu field value is the validation failure <ErrorType>
+    And SolutionDetail exist
+        | Solution | SummaryDescription             | FullDescription   | ClientApplication                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+        | Sln1     | An full online medicine system | Online medicine 1 | { "ClientApplicationTypes": [], "BrowsersSupported": [], "MobileResponsive": null, "Plugins": null, "MinimumConnectionSpeed": null, "MinimumDesktopResolution": null, "HardwareRequirements": null, "NativeMobileHardwareRequirements": null, "NativeDesktopHardwareRequirements": null, "AdditionalInformation": null, "MobileFirstDesign": null, "NativeMobileFirstDesign": null, "MobileOperatingSystems": null, "MobileConnectionDetails": null, "MobileMemoryAndStorage": null, "MobileThirdParty": null, "NativeMobileAdditionalInformation": null, "NativeDesktopOperatingSystemsDescription": null, "NativeDesktopMinimumConnectionSpeed": null } |
+Examples:
+        | ErrorType | FieldValue                  |
+        | maxLength | A string with length of 301 |
+        | required  |                             |
+        | required  | NULL                        |
+
```

Comment:
Maybe do another Scenario with multiple  invalid answers, so you can have several validation failures (line 63)